README.md

```diff
@@ -1,15 +1,29 @@
 # Espressif Github Actions
 
-[Github Actions](https://developer.github.com/actions/) developed/used by Espressif to help manage GitHub repos.
+[Github Actions](https://developer.github.com/actions/) developed by Espressif to help manage GitHub repositories.
 
-* `sync_issues_to_jira` performs one way syncing of GitHub issues into a JIRA project.
+### Sync issues to JIRA
 
-# Support and Changes
+`sync_issues_to_jira` performs one-way syncing of GitHub issues into a JIRA project.
 
-* Raising issues and sending Pull Requests is very welcome.
+### Upload components
 
-* However, please remember this repository exists to serve Espressif's internal GitHub integration needs. Issues or Pull Requests which aren't useful for those needs may not be addressed or merged. Sorry.
+`upload_components` uploads components from a GitHub repo to [Espressif Component Service](https://components.espressif.com)
```

Comment:
For `sync_issues_to_jira`, there is a one sentence summary in the top-level README here and then a full README in the subdirectory which has the config details, example of a workflow, etc.

I'd prefer to keep this pattern for all of the actions, as most of the information isn't shared (i.e. details of how to configure Action A are not relevant to someone using Action B), and the README will gradually get more cluttered if it has to include all details (a case of where this has happened is the esptool repo which started as a four line summary and five years later it's a mess!)

Putting README-per-directory also lets you directly link to the subdirectory URL for the action, and the README is rendered right there with it.

Reply:
Thank for the review and the suggestion, I moved details to the separate readme file